### PR TITLE
test fixes and annotators

### PR DESCRIPTION
**Pull request by YojanaGadiya**
The following changes happened in the PR:

- Fixing of the directory for test sets
- Adding in tests for OpenTargets and WikiData
- Adding an analyzer.explorer() function for literature and patent

### CODE REVIEW

**Comment by YojanaGadiya** [comment]
- Fixed Bgee error

File: tests/annotators/test_wikidata.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for the WikiData annotator."""
```
Comment (egonw) [inline tests/annotators/test_wikidata.py:4]: Wikidata (lower case 'd')

File: tests/annotators/test_wikidata.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import pandas as pd
+from numpy import nan
+
+from pyBiodatafuse.annotators import wikidata
+from pyBiodatafuse.annotators.wikidata import get_version_wikidata, get_gene_cellular_component
+from pyBiodatafuse.constants import WIKIDATA
+
+
+class TestWikidata(unittest.TestCase):
+    """Test the WikiData class."""
```
Comment (egonw) [inline tests/annotators/test_wikidata.py:18]: Wikidata (lower case 'd')

File: tests/annotators/test_wikidata.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+    # TODO: Fix this when the main function changes
+    def test_wikipathways_version(self):
```
Comment (egonw) [inline tests/annotators/test_wikidata.py:21]: error in method name

**Comment by YojanaGadiya** [comment]
- Fixing of Bgee speed. Removed development stage from annotation since this is always "adult" as per internal discussions.

**Comment by YojanaGadiya** [comment]
- Adding in literature and patent annotators

**Comment by tabbassidaloii** [comment]
@YojanaGadiya any update on this branch?

**Pull request merged**